README.md

```diff
@@ -1,17 +1,17 @@
-<p align="center">
-	<a href="https://eoscostarica.io">
-		<img src="https://github.com/eoscostarica/eos-rate/raw/master/docs/eoscostarica-logo-black.png" width="300">
+<div align="center">
+	<a href="https://edenia.com">
```

Comment:
link to https://eoscostarica.io not edenia.com

Reply:
fixed.